pkg/monitoring/metrics/virt-controller/vmstats_collector.go

```diff
@@ -210,7 +212,7 @@ func CollectVMsInfo(vms []*k6tv1.VirtualMachine) []operatormetrics.CollectorResu
 				vm.Name, vm.Namespace,
 				os, workload, flavor, machineType,
 				instanceType, preference,
-				string(vm.Status.PrintableStatus), getVMStatusGroup(vm.Status.PrintableStatus),
+				strings.ToLower(string(vm.Status.PrintableStatus)), getVMStatusGroup(vm.Status.PrintableStatus),
```

Comment:
Is the actual status, not  "PrintableStatus", not already in lower case?

Reply:
No, it's on big case:
![Screenshot from 2024-12-10 19-41-45](https://github.com/user-attachments/assets/d4b5b2a5-8f39-4189-be8e-6cbda072466a)